components/styles.js

```diff
@@ -44,17 +44,16 @@ const styles = StyleSheet.create({
   },
   dayButton: {
     alignItems: 'center',
-    padding: 7,
+    padding: 0,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Why do we need to specify the height here?

And if you really need to do this, you should do it to dayButtonFiller as well, since they both should be the same size.